views/js/portableLib/OAT/util/html.js

```diff
@@ -24,12 +24,6 @@ define([
 
     return {
         render : function render($container){
-            var markup = $container.html();
-
-            //remove xml ns (would break jquery selector otherwise)
```

Comment:
I'm wondering what does it mean, have you asked Sam or someone who could be aware of that?